### ISSUE TITLE
more_than_one_bad_input.test may fail due to timming

### ISSUE THREAD

**Issue by fgalan** [telefonicaid/fiware-orion#3828]
(Comes from https://github.com/telefonicaid/fiware-orion/pull/3825)

The more_than_one_bad_input.test sometime fails in some build systems (it has been detected in Raspberry) due to timming. Detail (from @fisuda feedback):
 
> The test logs are the followings:
> https://github.com/fisuda/report/tree/master/orion/20210325_centos8_aarch64
> 
> In the function test, one test has failed as shown:
> 
>    * 1697_more_than_one_bad_input
> 
> 
> This test may pass or fail. It seems that the issue is dependent on the timing. This behavior is the same as before (#3630).
> 
>    * more_than_one_bad_input.diff
> 
> 
> ```
> VALIDATION ERROR: input file and reference file have different line count
> ```
> 
>   * Expected (more_than_one_bad_input.regexpect)
> 
> 
> ```
> 3. grep SUMMARY in log-file (4 transactions, 1 bad input)
> ==========================================================
> Transactions: REGEX((4 _new: 4_|5 _new: 5_))
> DB status: ok, raised: _total: 0, new: 0_, released: _total: 0, new: 0_
> Notification failure active alarms: 0, raised: _total: 0, new: 0_, released: _total: 0, new: 0_
> Bad input active alarms: 1, raised: _total: 1, new: 1_, released: _total: 0, new: 0_
> ```
> 
>    * Actual (more_than_one_bad_input.out)
> 
> 
> ```
> 03. grep SUMMARY in log-file (4 transactions, 1 bad input)
> ==========================================================
> Transactions: 0 _new: 0_
> DB status: ok, raised: _total: 0, new: 0_, released: _total: 0, new: 0_
> Notification failure active alarms: 0, raised: _total: 0, new: 0_, released: _total: 0, new: 0_
> Bad input active alarms: 0, raised: _total: 0, new: 0_, released: _total: 0, new: 0_
> Transactions: 4 _new: 4_
> DB status: ok, raised: _total: 0, new: 0_, released: _total: 0, new: 0_
> Notification failure active alarms: 0, raised: _total: 0, new: 0_, released: _total: 0, new: 0_
> Bad input active alarms: 1, raised: _total: 1, new: 1_, released: _total: 0, new: 0_
> ```

**Comment by fgalan** [comment]
Maybe we can avoid the noise caused by "spurious" summary traces taking just the last 4 ones.

Could you test the following patch in your system, please?

```diff
--- a/test/functionalTest/cases/1697_more_than_one_bad_input/more_than_one_bad_input.test
+++ b/test/functionalTest/cases/1697_more_than_one_bad_input/more_than_one_bad_input.test
@@ -62,7 +62,7 @@ echo
 
 echo "03. grep SUMMARY in log-file (4 transactions, 1 bad input)"
 echo "=========================================================="
-egrep 'SUMMARY' /tmp/contextBroker.log | awk -F'|' '{ print $10 }' | awk -F= '{ print $2 }' | sed 's/(/_/g' | sed 's/)/_/g'
+tail -n 4 /tmp/contextBroker.log | grep 'SUMMARY' | awk -F'|' '{ print $10 }' | awk -F= '{ print $2 }' | sed 's/(/_/g' | sed 's/)/_/g'
 echo
 echo
 
```

If it works I'll create a PR to improve that test.

**Comment by fisuda** [comment]
I have tested the patch in my Raspberry Pi. It works fine! I encountered no errors in the functional tests.

```
0697/1159: 1693_preserve_sp_updating_subscription/preserve_sp_updating_subscription_ngsiv1.test .........................  06 seconds
0698/1159: 1693_preserve_sp_updating_subscription/preserve_sp_updating_subscription_ngsiv2.test .........................  06 seconds
0699/1159: 1697_more_than_one_bad_input/more_than_one_bad_input.test ....................................................  08 seconds
0700/1159: 1705_csub_cache_objects/csub_cache_objects_errors.test .......................................................  07 seconds
0701/1159: 1705_csub_cache_objects/csub_cache_string_filter_update.test .................................................  07 seconds

...etc

1159/1159: 3777_right_attr_filter_and_project/right_attr_filter_and_project.test ........................................  12 seconds
Total test time: 9794.64 seconds

WARNING: 5 test cases disabled:
  o  2: 0000_bad_requests/exit.test
  o  30: 0000_ipv6_support/ipv4_only.test
  o  31: 0000_ipv6_support/ipv6_only.test
  o  374: 0917_queryContext_behaves_differently/query_with_and_without_forwarding.test
  o  610: 1310_suspect_200OK/suspect_200OK.test
```

The following is the whole log:
https://github.com/fisuda/report/blob/master/orion/20210330_centos8_aarch64/test-log-20210330-0500.txt

**Comment by fgalan** [comment]
Having merged #3829 I understand this issue can be closed.
